src/Application.js

```diff
@@ -135,11 +149,35 @@ exports = Class(GC.Application, function () {
       })
     });
 
-    this.restoreButton = new ButtonView({
+    this.localizeButton = new ButtonView({
       superview: this.view,
       x: buttonPadding,
       y: this.purchaseButtonFail.style.y + 100,
-      width: buttonWidth + buttonWidth + buttonPadding,
+      width: buttonWidth,
+      height: 50,
+      title: "Localize Purchases",
+      onClick: bind(this, function () {
```

Comment:
Is there a reason we're creating a closure here instead of binding the function directly?